src/reducers/ActionWrapper.js

```diff
@@ -22,10 +22,6 @@ let actionWrapperReducer = (state = configData, action) => {
         ...state,
         is_app_loaded: true
       };
-    case actions.APP_FETCHING:
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it